pal/inc/pal_mstypes.h

```diff
@@ -200,7 +200,7 @@ extern "C" {
 // they must be either signed or unsigned) and we want to be able to use 
 // __int64 as though it were intrinsic
 
-#ifdef BIT64
+#ifndef BIT64
```

Comment:
> Also, on 32bit unix it would be wrong -- defining __int64 to long.

.....That's why this PR fixes __int64 for BIT64 and defines additionals only when BIT64 is set. So, no problem with 32 BIT or ARM...